.github/workflows/auto-close.yml

```diff
@@ -0,0 +1,25 @@
+name: Auto close bad PR
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+on:
+  pull_request_target:
+    branches:
+      - master
+      - '*-maintenance'
```

Comment:
Is this needed given the equality check to master later in the file?

Reply:
i followed the pattern on the other workflows.
if the default is any bracnh (`'*'`), then this could be only: 
```
pull_request_target:
```
(excluding `branches:` specifier)

Reply:
done. tested successful.